src/lib/litegraph/src/LiteGraphGlobal.ts

```
@@ -788,33 +795,30 @@ export class LiteGraphGlobal {
+    ]
+
+    if (pointerAndMouseEvents.includes(sEvent)) {
+      oDOM.addEventListener(sMethod + sEvent, fCall, capture)
+    }
+
+    if (
+      pointerAndMouseEvents.includes(sEvent) ||
+      pointerOnlyEvents.includes(sEvent)
+    ) {
+      if (sMethod != 'mouse') {
+        return oDOM.addEventListener(sMethod + sEvent, fCall, capture)
```

Biome (2.1.2)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)